include/crpropa/module/HDF5Output.h

```diff
@@ -82,7 +82,7 @@ class HDF5Output: public Output {
 		double P1x;
 		double P1y;
 		double P1z;
-		double weight;
+		double W;
```

Comment:
I would suggest to keep the double naming `weight` as it is easier to read. 
The change should only apply for the naming of the column of the final output file.